components/MapRoute.js

```diff
@@ -394,6 +403,7 @@ MapRoute.defaultProps = {
   showNavigationButton: true,
   isNavigation: false,
   initialFocus: 'gps',
+  trackNavigationEvent: false,
```

Comment:
Won't we log by default? I would expect `true` here.

Reply:
This is NextShiftMapScreen specific to track the event that the user is on its way to the interchange point. When the component is embedded somewhere else, it should be disabled by default, e.g. in RideScreen. Alternatively we could use a navigationStarted callback and do the logging in the NextShiftScreen?